src/Enemies.py

```diff
@@ -35,3 +35,23 @@ def __iadd__(self):
 class Ogre(Enemy):
     def __init__(self):
         super().__init__(name="Ogres", hp=15, damage=5, armor=1)
+
+
+class Dragon(Enemy):
+    def __init__(self):
+        super().__init__(name="Giant Spider", hp=25, damage=2, armor=4)
+
+
+class Wolf(Enemy):
+    def __init__(self):
+        super().__init__(name="Spiders", hp=6, damage=3, armor=0)
```

Comment:
This enemy should also be named wolve :P

Reply:
1 = will be changed after because keine lust jetzt